animation_nodes/nodes/gpencil/gp_stroke_from_points.py

```diff
@@ -1,39 +1,44 @@
 
 class GPStrokeFromPointsNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_GPStrokeFromPointsNode"
     bl_label = "GP Stroke From Points"
     errorHandlingType = "EXCEPTION"
 
-    useStrengthsList: VectorizedSocket.newProperty()
-    usePressuresList: VectorizedSocket.newProperty()
-    useUVRotationsList: VectorizedSocket.newProperty()
+    useStrengthList: VectorizedSocket.newProperty()
+    usePressureList: VectorizedSocket.newProperty()
+    useUVRotationList: VectorizedSocket.newProperty()
+    useVertexColorList: VectorizedSocket.newProperty()
 
     def create(self):
         self.newInput("Vector List", "Points", "vertices")
-        self.newInput(VectorizedSocket("Float", "useStrengthsList",
+        self.newInput(VectorizedSocket("Float", "useStrengthList",
             ("Strength", "strength"), ("Strengths", "strengths")), value = 1)
-        self.newInput(VectorizedSocket("Float", "usePressuresList",
+        self.newInput(VectorizedSocket("Float", "usePressureList",
             ("Pressure", "pressure"), ("Pressures", "pressures")), value = 1)
-        self.newInput(VectorizedSocket("Float", "useUVRotationsList",
+        self.newInput(VectorizedSocket("Float", "useUVRotationList",
             ("UV-Rotation", "uvRotation"), ("UV-Rotations", "uvRotations")), hide = True)
+        self.newInput(VectorizedSocket("Color", "useVertexColorList",
+            ("Vertex Color", "vertexColor", dict(value = (0, 0, 0, 0))), ("Vertex Colors", "vertexColors")))
         self.newInput("Float", "Line Width", "lineWidth", value = 250)
+        self.newInput("Float", "Hardness", "hardness", value = 1)
         self.newInput("Boolean", "Cyclic", "drawCyclic", value = False, hide = True)
         self.newInput("Text", "Start Cap Mode", "startCapMode", value = 'ROUND', hide = True)
         self.newInput("Text", "End Cap Mode", "endCapMode", value = 'ROUND', hide = True)
         self.newInput("Integer", "Material Index", "materialIndex", value = 0)
         self.newInput("Text", "Display Mode", "displayMode", value = '3DSPACE', hide = True)
         self.newOutput("GPStroke", "Stroke", "stroke")
 
-    def execute(self, vertices, strengths, pressures, uvRotations, lineWidth, drawCyclic,
-                startCapMode, endCapMode, materialIndex, displayMode):
+    def execute(self, vertices, strengths, pressures, uvRotations, vertexColors, lineWidth,
+                hardness, drawCyclic, startCapMode, endCapMode, materialIndex, displayMode):
 
         amount = len(vertices)
         strengths = FloatList.fromValues(VirtualDoubleList.create(strengths, 1).materialize(amount))
         pressures = FloatList.fromValues(VirtualDoubleList.create(pressures, 1).materialize(amount))
         uvRotations = FloatList.fromValues(VirtualDoubleList.create(uvRotations, 0).materialize(amount))
+        vertexColors = ColorList.fromValues(VirtualColorList.create(vertexColors, Color((0, 0, 0, 0))).materialize(amount))
```

Comment:
The output of `materialize` is already a ColorList, so no need for `ColorList.fromValues` here.

Reply:
Okay.

Reply:
Done.